File: dcli/template/java/src/main/java/template/Target.java
package template;

import java.util.*;

public class {{ targetFile }} {

    /**
     * Called when you Run your code
     */
    public static void main() {

    }

    /**
     * Called when you Submit your code
     */
    public static {% if targetGenerics != "" %}<{{ targetGenerics }}> {% endif %}{% if targetMethodReturn == "void" %}void {% else %}{{ targetMethodReturn }}{% endif %} {{ targetMethod }}({{ targetMethodArgs }}) {
        //TODO write your own solution
        return {% if targetDefaultReturn == None %} null {% else %}{{ targetDefaultReturn }}{% endif %};
    }
}

File: dcli/template/kotlin/src/main/kotlin/template/Target.kt
package template

class {{ targetFile }} {
    companion object {
        /**
        * Called when you Run your code
        */
        fun main() {

        }

        /**
        * Called when you Submit your code
        */
        fun {{ targetMethod }}({{ targetMethodArgs }}): {% if targetMethodReturn == "" %}Unit{% else %}{{ targetMethodReturn }}{% endif %} {
            {% if targetMethodReturn != "" %}return {{ targetMethodReturnValue }};{% endif %}
        }
    }
}